### ISSUE TITLE
"make dev" fails when calling test_data.py 

### ISSUE THREAD

**Issue by loadstar81** [lucyparsons/OpenOversight#694]
It looks like this went from a deprecation warning to a hard error when pytest was upgraded from 4.0.2 --> 5.2.2 [e1393a2e](https://github.com/lucyparsons/OpenOversight/commit/e1393a2e413fc7dc4410e6a6102c2ef53750164f#diff-913fedaed3cb05fe362c0ceb35e2d827)
pytest issue in 4.1.0 enforcing this behavior: [#4545](https://github.com/pytest-dev/pytest/issues/4545)
```
## Populate database with test data
docker-compose run --rm web python ../test_data.py -p
Starting openoversight_postgres_1 ... done
[2019-12-20 20:04:25,179] INFO in __init__: OpenOversight startup
[2019-12-20 20:04:25,436] INFO in __init__: OpenOversight startup
[*] Populating database with test data...
Traceback (most recent call last):
  File "../test_data.py", line 25, in <module>
    mockdata(db.session)
  File "/ve/lib/python3.7/site-packages/_pytest/fixtures.py", line 978, in result
    fail(message, pytrace=False)
  File "/ve/lib/python3.7/site-packages/_pytest/outcomes.py", line 128, in fail
    raise Failed(msg=msg, pytrace=pytrace)
Failed: Fixture "mockdata" called directly. Fixtures are not meant to be called directly,
but are created automatically when test functions request them as parameters.
See https://docs.pytest.org/en/latest/fixture.html for more information about fixtures, and
https://docs.pytest.org/en/latest/deprecations.html#calling-fixtures-directly about how to update your code.
make: *** [populate] Error 1
```
A current workaround is to revert pytest to version 4.0.2

**Comment by abandoned-prototype** [comment]
Thanks for bringing this up @loadstar81. I just ran into that issue too and wasn't sure if this is something locally I messed up, or broken for everyone. Reverting to 4.0.2 works for me

**Comment by ssempervirens** [comment]
Hi, I reverted pytest to 4.0.2, but `make dev` is still failing for me. Specifically, I am getting:
````
sqlalchemy.exc.IntegrityError: (psycopg2.errors.UniqueViolation) duplicate key value violates unique constraint "ix_departments_name"
DETAIL:  Key (name)=(Springfield Police Department) already exists.
 [SQL: 'INSERT INTO departments (name, short_name, unique_internal_identifier_label) VALUES (%(name)s, %(short_name)s, %(unique_internal_identifier_label)s) RETURNING departments.id'] [parameters: {'name': 'Springfield Police Department', 'short_name': 'SPD', 'unique_internal_identifier_label': 'homer_number'}]
make: *** [populate] Error 1
````

**Comment by redshiftzero** [comment]
fixed in #702
